Configuration/StandardSequences/python/Eras.py

```diff
@@ -61,6 +61,7 @@ def __init__(self):
                            'run2_nanoAOD_94XMiniAODv1', 'run2_nanoAOD_94XMiniAODv2', 'run2_nanoAOD_94X2016',
                            'run2_miniAOD_devel', 'run2_nanoAOD_102Xv1',
                            'hcalHardcodeConditions', 'hcalSkipPacker',
+                           'run2_zcount_2016','run2_zcount_2017','run2_zcount_2018',
```

Comment:
This kind of additions are going to be needed as a general modification to treat Trigger paths in HLT for the Run2 years. Perhaps a more general name like "run2_HLTconditions_201X"  could work for everybody otherwise we could end having every PAG/POG adding its own Modifier for the same use case: changes in HLT paths.
Just my two pence on this issue